src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
```

Comment:
The class Circle shouldn't start with an empty line. Please remove the empty line before the class declaration.